### ISSUE TITLE
Should this repo be archived and deactivated?

### ISSUE THREAD

**Issue by durack1** [PCMDI/nominal_resolution#8]
We have a CDAT dependence in here, and so this is another candidate for deactivation/archival

@mauzey1 @taylor13

**Comment by taylor13** [comment]
No.  Not unless the code and [documentation](https://pcmdi.github.io/nominal_resolution/html/) is available elsewhere.  We point to this code from, for example, https://wcrp-cmip.github.io/WGCM_Infrastructure_Panel/Papers/CMIP6_global_attributes_filenames_CVs_v6.2.7.pdf .  I think this code is used by CMIP modelers, and I have long had plans to summarize resolution information in a paper.   (But who knows if I'll get to that.). Besides that, it's a cool bit of coding.

**Comment by taylor13** [comment]
I have been getting weekly notices, which maybe we should do something about:  

![Screenshot 2024-06-11 at 12 58 18 PM](https://github.com/PCMDI/nominal_resolution/assets/4993439/38c6ad9c-f615-4152-ba86-06a284fa542b)

**Comment by durack1** [comment]
Yeah exactly, that is what triggered me too - this could definitely be useful, would be curious to know how much time it'd take to "upgrade" to supported libraries?

**Comment by taylor13** [comment]
I don't think the [code itself](https://github.com/PCMDI/nominal_resolution/blob/master/lib/api.py) needs CDAT.  

Here are its dependencies:
```
from __future__ import print_function
import numpy
import numpy.ma
import warnings
```

There are some [test codes](https://github.com/PCMDI/nominal_resolution/tree/master/tests) also.  Not sure where the problematic libraries are needed/used.

**Comment by durack1** [comment]
It might be an easier lift than we think, I do see some old libraries in https://github.com/PCMDI/nominal_resolution/blob/master/run_tests.py, but you're right, dependencies seem light
